gr-trellis/examples/python/test_tcm.py

```diff
@@ -112,7 +112,7 @@ def main():
         terr_p=terr_p+(e!=0)
         if ((i+1)%100==0) : # display progress
             print(i+1,terr_p, '%.2e' % ((1.0*terr_p) / (i+1)),tot_b,terr_b, '%.2e' % ((1.0*terr_b) / tot_b))
-	if e!=0:
+        if e!=0:
```

Comment:
This looks like a syntax / program flow change. Was that intended?

Reply:
if it is a real change no, but most importantly this piece of code does not run without fixing the mixed tab/space indentation (python will even refuse to import the file)